src/applications/static-pages/libraries/library-pages.js

```diff
@@ -0,0 +1,20 @@
+import { libraryProcess } from './library-process';
+import * as Bucket from '../../../../src/site/constants/buckets';
+import * as Environment from '../../../../src/platform/utilities/environment';
+/**
+ * For adding library pages.
+ */
+export function libraryPages() {
+  const outReachMaterials =
+    Environment.default.BUILDTYPE === 'vagovprod'
+      ? `${Bucket.vagovprod}/generated/outreach-assets.json`
+      : `${Bucket.vagovstaging}/generated/outreach-assets.json`;
+
+  const path = window.location.pathname.split('/');
+
+  if (path.includes('public-outreach-materials')) {
+    libraryProcess(outReachMaterials);
```

Comment:
Since this is a full-page app and not a widget, I am really wondering if there's any way for us to define this work with a separate JavaScript entry point. I think it should be relatively easy, and then we can possibly save us some extra trouble by writing data to the page instead of fetching it in an HTTP request.

Is `src/site/layouts/office.drupal.liquid` a layout dedicated to the work with outreach assets?